client/templates/admin/inc/sys_header.html

```diff
@@ -45,7 +45,7 @@
               </li>
             {{/if}}
             {{#if isInRole 'admin' 'CB'}}
-              {{> loginButtons align="right"}}
+              <!-- {{> loginButtons align="right"}} -->
```

Comment:
@distalx Quick question -- why did you comment out these login buttons for admin CB users?